uui/components/tables/DataTableCell.tsx

```diff
@@ -33,6 +33,10 @@ function DataTableRowAddons<TItem, TId, TCellValue>(props: DataTableCellProps<TI
                 <div key="fold" className={ css.indent } style={ { marginLeft: (row.indent - 1) * 24 } }>
                     {row.isFoldable && (
                         <IconContainer
+                            rawProps={ {
+                                'aria-label': 'Fold/Unfold',
+                                'data-testid': `uui-DataTableRowAddons-folding-arrow-${row.rowKey}`,
```

Comment:
Fold/unfold can be in different levels, so, adding `data-testid` will make it possible to query not all the folding arrows, but the exact one. And this pattern of querying something by role and aria-label is kind of asking if somebody knows the person with long hair and blue eyes, and how is working as an engineer, instead of asking if somebody knows the person with the exact name. WDYT?

Reply:
The difference is what is tested: "implementation details" (data attributes) vs "end result" (can be visible or perceivable by end user). So that test fails only when visible behavior changes, and doesn't fail if only internal details change. Same is recommended by the testing library docs https://testing-library.com/docs/queries/about/#priority

I don't know how to find this button at a specific nesting level. Maybe query by item label first, and then query by Fold/Unfold label in same container. If it's impossible or really difficult, then it's ok to leave approach with data-testid